services/workflows-service/src/customer/types.ts

```diff
@@ -1,7 +1,35 @@
+import { Customer } from '@prisma/client';
+
 export type TAuthenticationConfiguration = {
   apiType: 'API_KEY' | 'OAUTH2' | 'BASIC_AUTH';
   authValue: string;
   validUntil?: string;
   isValid: boolean;
   webhookSharedSecret: string;
 };
+
+export const CUSTOMER_FEATURES = {
+  ONGOING_AUDIT_REPORT: {
+    name: 'ONGOING_AUDIT_REPORT',
+    definitionName: 'ongoing_audit_report',
+  },
+} as const;
+
+export type TCustomerFeatures = {
+  name: keyof typeof CUSTOMER_FEATURES;
+  enabled: boolean;
+  definitionName?: string;
+};
+
+export type TOngoingAuditReportDefinitionConfig = {
+  definitionName: string;
+  intervalInDays: number;
+  active: boolean;
+  checkType: string[];
+  proxyViaCountry: string;
+};
+
+export type TCustomerWithDefinitionsFeatures = Partial<Customer> & {
+  features?: TCustomerFeatures[] | null;
+  definitionConfigs?: TOngoingAuditReportDefinitionConfig[] | null;
```

Comment:
It shouldn't be called definitionConfigs and have a separate column, customers already have a config object and it should have a configuration about ongoing setups (whether its kinds business monitoring, sanctions, IDV etc ) - definitionId is just another asset in the ongoing configuration object - assume that other services is using this configuration where its not related to the definition so it doesn't make sense to couple it in that way to a definition where its a way to execute a check but it might also be in the future an API request for a status from 3rd party.